zoo-css-challenge/style.css

```
@@ -29,13 +30,14 @@ section {
   background-color: #f8f0fb;
 }
 
-img {
-  border: 5px solid #8075ff;
+#logo {
+  border: none;
+}
+#logo {
 }
-
 h1,
-h2,
-h3,
+/*h2,
+h3,*/
```

```suggestion
/*h2,
h3,*/
```
We shouldn't leave commented code. We also don't need to take out `h2` and `h3` in this exercise